waffle/tests/test_templates.py

```diff
@@ -35,8 +35,11 @@ def test_django_tags(self):
         self.assertContains(response, 'window.waffle =')
 
     def test_get_nodes_by_type(self):
-        """WaffleNode.get_nodes_by_type() should correctly find all child nodes"""
-        test_template = Template('{% load waffle_tags %}{% switch "x" %}{{ a }}{% else %}{{ b }}{% endswitch %}')
+        """WaffleNode.get_nodes_by_type() should find all child nodes."""
+        test_template = Template(
+            '{% load waffle_tags %}{% switch "x" %}{{ a }}'
```

Comment:
nit but can you break this into a few lines? i think it'll be a _lot_ more readable

```py
'{% load waffle_tags %}'
'{% switch "x" %}'
'... etc ...'
```